ext/linguist/tokenizer.l

```diff
@@ -102,15 +102,17 @@
 "\"\"\""                 { FEED_STATIC("COMMENT\"\"\""); BEGIN(python_dcomment); return 1; }
 "'''"                    { FEED_STATIC("COMMENT'''"); BEGIN(python_scomment); return 1; }
 ^".ig"\n                 { FEED_STATIC("COMMENT.ig"); BEGIN(roff_comment); return 1; }
+"/-"                     { FEED_STATIC("COMMENT/-"); BEGIN(lean_comment); return 1; }
```

Comment:
This is lex. The current `/-` already matches `/--` so I don't think we need to add `/--` too.

Reply:
We need to match `/--` as a separate token if we want to correctly parse `/--/ -/` as a single comment containing `/ `, otherwise the trailing `-` is consumed as a close comment token.

Reply:
Ah. I see now. https://github.com/github-linguist/linguist/pull/6625/commits/c27ac2f4f70c0e0a4cd57be16cc65665caf60049 should do the trick